recipes/kraken-all/meta.yaml

```diff
@@ -0,0 +1,13 @@
+package:
+  name: kraken-all
+  version: "0.10.5beta"
+requirements:
+  build:
+    - kraken ==0.10.5beta
+  run:
+    - kraken
+    - jellyfish >=1.1.11,<2
+about:
+  home: http://ccb.jhu.edu/software/kraken/
+  license: GPLv3
+  summary: Kraken is a system for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies.
```

Comment:
Maybe the summary should explain in what sense this is different to the kraken package.